index.html

```diff
@@ -26,7 +26,7 @@
                     <li><a href="pages/frontend/work.html">work</a></li>
                     <li><a href="pages/frontend/team.html">team</a></li>
                     <li><a href="pages/frontend/blog.html">blog</a></li>
-                    <li><a href="#">contact</a></li>
+                    <li><a href="pages/frontend/blog.html">contact</a></li>
```

Comment:
Change this to contact instead
```suggestion
                    <li><a href="pages/frontend/contact.html">contact</a></li>
```

Reply:
My bad. Done